packages/opentelemetry-instrumentation-replicate/tests/test_import.py

```
@@ -0,0 +1,6 @@
+import sys
+
+def test_import():
+    import opentelemetry.instrumentation.replicate
```

Ruff (0.12.2)

`opentelemetry.instrumentation.replicate` imported but unused

Remove unused import: `opentelemetry.instrumentation.replicate`

(F401)